packages/grpc-native-core/index.d.ts

```diff
@@ -9,7 +9,7 @@ declare module "grpc" {
    * @param options Options to apply to the loaded file
    * @return The resulting gRPC object.
    */
-  export function loadObject(value: object, options?: LoadObjectOptions): GrpcObject;
+  export function loadObject<T = GrpcObject>(value: object, options?: LoadObjectOptions): T;
```

Comment:
I don't understand this change. The output of this function can only be `GrpcObject` as defined below. What does parameterizing it accomplish?

Reply:
See below.